bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: [generic template refresh 1] Change kafka references to dinosaurs

## Description

This is the first change of a set of changes needed to get all patches from generic_template up to date with main. See https://issues.redhat.com/browse/MGDSTRM-7041 comments for details.

This PR changes updates the content to change kafka-related references to dinosaur-related references.

You will see there are some edge cases if you look for 'kas' in the code (grep -R -i kas *). Migrations are also not consistent because they will be dealt in a future PR.

This PR is done against the `refresh_generic_template_content` branch.

See the commit descriptions for details on what each commit does.

The PR also contains a fix that is happening in the main branch. a  separate PR will be opened to include the fix there too.

## Verification Steps

1. clone the repo
2. run `make openapi/generate` - make sure that no new files are generated
3. run `make run` (db setup and secrets setup will be required due to renaming of them) - make sure that the service starts. It will produce some erros in the workers, but this will be fixed in the subsequent Jiras (db schema, workers, tests, etc)
4. Run `find . -type f -name '*kafka*'` and make sure that there are no files with *kafka* in their name (apart from migrations which will be handled in separate Jira).
5. Check for occurrences of "kas-fleet-manager", "kas_fleet_manager", "kafka", "KAFKA", "Kafka" and verify there are none. As it's been commented above there are some edge cases if you look for 'kas' in the code

## Checklist (Definition of Done)
